### PR TITLE
[uss_qualifier/webapp] Open api implementation of `test_runs/` endpoint

### PR DESCRIPTION
**Pull request by pratibhagupta2109**

### CODE REVIEW

File: interfaces/uss_qualifier/uss_qualifier.yaml [hunk 1]
```diff
@@ -1,198 +1,272 @@
...
+                        Content of test runs' json output file.
+            required:
+                - user_id
+                - specification
+
+        TestsResponse:
+            type: object
+            properties:
+                test_runs:
+                    type: object
```
[the remark this answers is not in the document]
Comment (pratibhagupta2109) [reply interfaces/uss_qualifier/uss_qualifier.yaml:115]: It is an object containing `test_runs` as a key and test_runs details (filename: report) as value.

File: interfaces/uss_qualifier/uss_qualifier.yaml [hunk 2]
```diff
@@ -1,198 +1,272 @@
...
+                    example: "A task has been started in the background."
+                    description: >-
+                        Status of currently running task.
+                files:
+                    type: array
+                    items:
+                        type: string
+                    description: >-
+                        KML files uploaded to generate flight records.
+                background_tasks:
```
[the remark this answers is not in the document]
Comment (pratibhagupta2109) [reply interfaces/uss_qualifier/uss_qualifier.yaml:62]: The part to get generated flight record filenames are not returned by task_id yet. I will add that to /tasks/<task_id> response.

File: interfaces/uss_qualifier/uss_qualifier.yaml [hunk 3]
```diff
@@ -1,198 +1,272 @@
...
+                  required: true
+                  description: An output file name for one of the test runs.
+                  schema:
+                      $ref: "#/components/schemas/TestRunID"
+            responses:
+                "200":
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/TestsResponse"
```
[the remark this answers is not in the document]
Comment (pratibhagupta2109) [reply interfaces/uss_qualifier/uss_qualifier.yaml:194]: yes. It needs to be corrected.

File: interfaces/uss_qualifier/uss_qualifier.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pratibhagupta2109) [reply interfaces/uss_qualifier/uss_qualifier.yaml:62]: Getting the generated flight record filenames is not straightforward  through TaskIdStatusResponse. 
Task completes at redis server and retains the flight records in json format. It is the uss-host, which pull this task result from redis server and `evaluate` which filename should be given to the flight record based on the previous available versions of that filename.

File: interfaces/uss_qualifier/uss_qualifier.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pratibhagupta2109) [reply interfaces/uss_qualifier/uss_qualifier.yaml]: Converted to an array.

File: interfaces/uss_qualifier/uss_qualifier.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pratibhagupta2109) [reply interfaces/uss_qualifier/uss_qualifier.yaml:62]: updated api/tasks/<task_id> to show generated_flight_records if task is from kml_import_jobs.

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**